### PR TITLE
Add non-ASCII logs check task in master dag

### PR DESCRIPTION
**Pull request by vatsrahul1001**
This PR is to test the astronomer logging provider for non-ASCII character, as we have providers tests running on all cloud providers daily, This will help us test daily without creating additional deployments.

### CODE REVIEW

File: .circleci/integration-tests/master_dag.py [hunk 1]
```diff
@@ -199,7 +199,20 @@ def prepare_dag_dependency(task_info, execution_time):
         ],
     )
 
-    airflow_version_check = (get_airflow_version, check_logs_data)
+    # The below task `check_non_ascii_data` is specifically added to test the astronomer logging
+    # providers for non-ASCII characters, as we have providers' tests running on all
+    # cloud providers, and we do not want multiple deployments for this.
+    check_non_ascii_data = PythonOperator(
+        task_id="check_non_ascii_data",
+        python_callable=check_log,
```
Comment (pankajkoti) [inline .circleci/integration-tests/master_dag.py:207]: Where is this callable check_log defined?

**Review by pankajkoti — COMMENTED** [review]
I pushed a commit to transform the comment from docstring to comment.

File: .circleci/integration-tests/master_dag.py (hunk 1 above)
Comment (vatsrahul1001) [reply .circleci/integration-tests/master_dag.py:207]: we were already using this callable from [here](https://github.com/astronomer/astronomer-providers/blob/a49fe81d86ae49e4ddba7ae988914d703e79c05f/.circleci/integration-tests/airflow_dag_introspection.py#L36)

**Review by pankajkoti — APPROVED** [review]

File: .circleci/integration-tests/master_dag.py (hunk 1 above)
Comment (pankajkoti) [reply .circleci/integration-tests/master_dag.py:207]: sorry forgot to check the imports.

**Pull request merged**